tests/test_download.py

```diff
@@ -0,0 +1,73 @@
+import uuid
+import pytest
+import json
+
+from tempfile import TemporaryDirectory
+
+from gtfs_rt_validator_api import download_gtfs_schedule_zip, download_rt_files, validate, validate_gcs_bucket
+from calitp.storage import get_fs
+from pathlib import Path
+
+GCS_BASE_DIR="gs://calitp-py-ci/gtfs-rt-validator-api"
```

Comment:
It would be helpful to add a comment noting the existence of the `seed_bucket_data.py` script that can help add data testing data to this directory.

Reply:
done, thanks!